deploy/deploy.sh

```diff
@@ -50,17 +49,24 @@ declare -a application_list=(
     'service-assurance/prometheus/service_account.yaml'
     'service-assurance/prometheus/role.yaml'
     'service-assurance/prometheus/rolebinding.yaml'
-    'service-assurance/prometheus/service_monitor.yaml'
     'service-assurance/prometheus/prometheus.yaml'
     'service-assurance/prometheus/route.yaml'
     'service-assurance/prometheusrules/prometheusrules.yaml'
     'service-assurance/alertmanager/service_account.yaml'
     'service-assurance/alertmanager/secret.yaml'
     'service-assurance/alertmanager/alertmanager.yaml'
-    'service-assurance/alertmanager/service.yaml'
     'service-assurance/alertmanager/route.yaml'
 )
 
+declare -a crds_to_wait_for=(
+    'alertmanagers.monitoring.coreos.com'
+    'prometheuses.monitoring.coreos.com'
+    'prometheusrules.monitoring.coreos.com'
+    'qdrs.interconnectedcloud.github.io'
```

Comment:
In the future this will change to `interconnects` I believe. No change for now, just noting it.